### PR TITLE
 I set the 0 , which allows to select all the backups.

### PR DESCRIPTION
**Pull request by SamirPS**

### CODE REVIEW

File: AstroSaveConverter.py [hunk 1]
```diff
@@ -1,182 +1,187 @@
-import argparse
```
Comment (Tignus) [inline AstroSaveConverter.py:1]: Pull request to be redone so that the diff can be seen properly.

File: AstroSaveConverter.py [hunk 2]
```diff
@@ -1,182 +1,187 @@
...
+    Returns:
+        Returns "path" if it was non-empty
+        Else returns the chosen container
+
+    Exception:
+        Raise FileNotFoundError if no container is found
+    """
+    list_containers = []
+    
+    if path==' ' or path==None:
```
Comment (Tignus) [inline AstroSaveConverter.py:41]: Format

File: AstroSaveConverter.py [hunk 3]
```diff
@@ -1,182 +1,187 @@
...
+
+        AstroLogging.logPrint('\nContainers found:')
+        for i, container in enumerate(list_containers):
+            AstroLogging.logPrint(f'\t {i+1}) {container}')
+
+    min_container_number = 1
+    max_container_number = len(list_containers)
+    path_index = 0
+
+    while not path_index :
```
Comment (Tignus) [inline AstroSaveConverter.py:57]: path_index is an int, not a boolean. Code rejected

File: AstroSaveConverter.py (hunk 2 above)
Comment (Tignus) [inline AstroSaveConverter.py:41]: Why check for blank space and not empty string ?

File: AstroSaveConverter.py (hunk 3 above)
Comment (SamirPS) [reply AstroSaveConverter.py:57]: for python 0 is false and another int it's true

File: AstroSaveConverter.py (hunk 2 above)
Comment (SamirPS) [reply AstroSaveConverter.py:41]: sorry it's was path==''

File: AstroSaveConverter.py (hunk 3 above)
Comment (Tignus) [reply AstroSaveConverter.py:57]: Nothing to do with Python here, we'll discuss that directly

File: AstroSaveConverter.py (hunk 2 above)
Comment (Tignus) [reply AstroSaveConverter.py:41]: (Please use pep-8)

**Comment by Tignus** [comment]
Pull request closed.
Not merged because of the quality.

**Pull request closed without merging**